storage/mysql/map_storage.go

```diff
@@ -73,7 +73,7 @@ func (m *mySQLMapStorage) CheckDatabaseAccessible(ctx context.Context) error {
 }
 
 type readOnlyMapTX struct {
-	tx *sql.Tx
+	*sql.Tx
```

Comment:
I think this OK because the code is MySQL specific and doesn't need to do anything special on commit. That might not be true on other databases but we don't need to worry about it here.